### DISCUSSION TITLE
Future of Rustdesk - AdressBook etc.

### DISCUSSION THREAD

**Discussion by bloesss** [rustdesk/rustdesk#4426] [General]
Hi,

I saw that you finally got an AddressBook working - even if it is beta.
What about your future plans? Will the AdressBook be a free feature for self hosted clients?
What features will be paid only?

I am willing to pay. But if we start now with rustdesk, we want to know if and what we have to pay for.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server/issues/158#issuecomment-1545385247

**Reply by bloesss** [reply]
Thank you, but where can I find an answer to my questions there?
The question is: What features will I have to pay for. And how much?
